controllers/admin/AdminModulesController.php

```diff
@@ -950,7 +951,7 @@ public function postProcessCallback()
                             if (isset($this->_modules_ad[$module->name])) {
                                 $ad_modules = $this->getModulesByInstallation($this->_modules_ad[$module->name]);
 
-                                foreach ($ad_modules['not_installed'] as $key => &$module) {
+                                foreach ($ad_modules['not_installed'] as &$module) {
```

Comment:
This foreach if part of another one, and its value used everywhere was erased by this one.
l.702, you can read `foreach ($this->map as $key => $method) {`.